docs/yarn.md

```diff
@@ -127,6 +127,12 @@ document has been completed, then it's essentially just a matter of cachi2 inter
 ``yarn install --mode=skip-build`` to fetch all dependencies (including transitive dependencies).
 
 ### Known pitfalls
+
+If your sources were not fetched from a remote git repo, Cachi2 will get confused about the name of 
+the main package, throwing an error: ``InvalidGitRepositoryError for file: URL``. As workaround, run 
+``git init; git remote add origin orgin https://github.com/somewhere/out-there/`` so that ``git remote show origin`` will return a value
```

Comment:
actually it turns out also an initial commit is needed as well, otherwise the command throws this error:

```text
2025-03-06 19:13:39,975 ERROR ValueError: Reference at 'refs/heads/main' does not exist
```

Reply:
workaround:
```
git init && git add -A && git commit -m "initial commit" && git remote add origin https://github.com/someorg/somerepo
```